display-posts-shortcode-remote.php

```diff
@@ -227,10 +232,10 @@ protected function cacheKey( $url ) {
 		 * @return array|WP_Error
 		 */
 		public function getPosts( $untrusted ) {
-
+			//Category Id was disabled in order to list all the posts available.  
 			$defaults = array(
 				'url'           => '',
-				'category_id'   => 0,
+				// 'category_id'   => 0,
```

Comment:
This should default to an empty string. There's a check a few lines down to see if it's _not_ empty. So if it's empty, that block of code will be skipped.

```suggestion
				'category_id'   => '',
```